install-laravel.sh

```diff
@@ -12,18 +12,44 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
-docker run --rm -v "$(pwd)":/opt -w /opt --platform=linux/amd64 laravelsail/php80-composer:latest bash -c "laravel new university && cd university && php ./artisan sail:install --with=redis,meilisearch,mailhog,selenium "
+# Create Laravel project
+docker run --rm -v "$(pwd)":/opt -w /opt --platform=linux/amd64 laravelsail/php80-composer:latest bash -c "laravel new university && cd university && php ./artisan sail:install --with=selenium "
 
 cd university
 
-echo ""
-
+# Make sure the user owns all of the new files
 sudo chown -R $USER: .
 
+# Install Tailwind CSS
+npm install
+npm install -D tailwindcss
+npx tailwindcss init
+echo "@tailwind base;" >> resources/css/app.css
+echo "@tailwind components;" >> resources/css/app.css
+echo "@tailwind utilities;" >> resources/css/app.css
+sed -i 's/^        \/\//        require(\d39tailwindcss\d39),/' webpack.mix.js
+
+# Copy welcome HTML to default Laravel view, customize
+cp ../web/index.html resources/views/welcome.blade.php
+cp ../web/assets/* public/
+sed -i 's/<link href="assets\//<link href="/' resources/views/welcome.blade.php
+sed -i '/<link/a\        <link rel="stylesheet" href="/css/app.css" />' resources/views/welcome.blade.php
```

Comment:
Why the "a" in `'/<link/a\     `?
Also - for slight readability, you can use something other then / as the argument separator in sed. for example, you could use `:`, so this line could be written as:

`sed -i ':<link:a\        <link rel="stylesheet" href="/css/app.css" :>' resources/views/welcome.blade.php`

Reply:
The intention of that line is to append (a) a new link tag after the existing link tag

Reply:
🤦 Right. My brain defaults to "sed is used to switch".